javascript-bank/bank.js

```diff
@@ -0,0 +1,44 @@
+/* exported Bank */
+
+function Bank() {
+  this.nextAccountNumber = 1;
+  this.accounts = [];
+}
+
+Bank.prototype.openAccount = function (holder, balance) {
+  if (balance > 0) {
+    var acct = new Account(this.nextAccountNumber, holder);
+    acct.deposit(balance);
+    this.accounts.push(acct);
+    this.nextAccountNumber++;
+    return acct.number;
+  } else {
+    return null;
+  }
+};
+
+Bank.prototype.getAccount = function (number) {
+  var match = null;
+  for (var i = 0; i < this.accounts.length; i++) {
+    if (this.accounts[i].number === number) {
+      match = this.accounts[i];
+    }
+  }
+  if (match === null) {
+    return null;
+  } else {
+    return match;
+  }
+};
+
+Bank.prototype.getTotalAssets = function () {
+  var total = 0;
+  if (this.accounts.length < 0) {
+    return 0;
```

Comment:
This can be removed. If there is no length to the array, the loop will never start, which leaves you with the same result.

Reply:
Done.